src/components/UserInfo.js

```diff
@@ -31,6 +32,7 @@ class UserInfo extends React.Component {
                         <p>Gus Username: {data.gus_username}</p>
                         <p>Gus ID: {data.gus_id}</p>
                         <p>Codes Redeemed: {data.number_of_redeems}</p>
+                        <p>Current Balance: {Accounting.formatMoney(data.current_balance_in_cents / 100)}</p>
```

Comment:
@lamngue this line is good. But can you also modify the line 17 in this same file to use `Accounting.formatMoney` instead of what we have right now? Thanks